### DISCUSSION TITLE
Is it possible to make custom NER model to optimize on precision rather than accuracy?

### DISCUSSION THREAD

**Discussion by aliasneo1** [explosion/spaCy#8254] [Help: Other Questions] [answered]
My current custom spacy NER model(V3.0) is good on recall but I want to improve  precision rather than recall. Is it possible to set it in config file?

- Operating System: Windows
- Python Version Used: 3
- spaCy Version Used: 3
- Environment Information:

**Comment by Tadusko** [comment, marked as answer]
Yes! I believe you need to change the values in `[training.score_weights]` section of the config file and train with the new weights. These scores define what SpaCy believes to be the best model after the training process.

By default, the bloc might look something like:
```
[training.score_weights]
ents_per_type = null
ents_f = 1.0
ents_p = 0.0
ents_r = 0.0
```
To give precision more weight, I suppose something like this could work:
```
[training.score_weights]
ents_per_type = null
ents_f = 0.0
ents_p = 0.75
ents_r = 0.25
```
More info on metrics and scoring [in the documentation](https://spacy.io/usage/training#metrics)

**Reply by aliasneo1** [reply]
Thanks. This is what exactly I am looking for.
